python/lsst/drp/tasks/assemble_coadd.py

```diff
@@ -138,10 +140,14 @@ def __init__(self, *, config=None):
 class AssembleCoaddConfig(
     CoaddBaseTask.ConfigClass, pipeBase.PipelineTaskConfig, pipelineConnections=AssembleCoaddConnections
 ):
-    warpType = pexConfig.Field(
+    warpType = pexConfig.ChoiceField(
```

Comment:
I don't think this breaks backwards compatibility at all. Trying to read in old config files would just work fine, as long as the values they have are one of the allowed values. These are the really two ones we support, and if the value was anything else, only then would it break (and it would be real error anyway). One way to see it is that we now have an extra constraint in the `validate` method, except that the check would happen much earlier.